### ISSUE TITLE
Suggestion: Save coords

### ISSUE THREAD

**Issue by PDeception** [finndev/PokeBuddy#191]
When I have to restart the application, or even just hit the Stop and Start button, it flings my guy back to the starting coords I gave it. Thus, softbanning me if its far enough. A feature where it saves your most recent coords and just uses them upon start-up would be nice.

**Comment by lordofthereef** [comment]
Have you actually been softbanned for this or is it just a theoretical? I noticed the bot sometimes wanders off but it hasn't ever been THAT bad!

**Comment by curtiola** [comment]
#172

**Comment by clarkeybear** [comment]
I have been victim of soft-banned multiple times due to my player wandering to far from my starting coords. Currently I have to re-enter updated coords so I prevent soft-banning myself. This could be a useful feature.

**Comment by curtiola** [comment]
yeah i totally agree, its on the suggestions list i referenced above @#172
also if you use these coordinates: 40.77844539228035,-73.96852254867554 central park NY, he almost never goes far enough away to get a soft ban, and he always does the same loop, its a long loop but it doesn't vary much, if at all

**Comment by pokemonbotacc** [comment]
how do i know if im soft banned?

**Comment by Vanethir** [comment]
@pokemonbotacc You get the message **[Farm Pokestops] Success -> Received XP: 0**

**Comment by PDeception** [comment]
Yes, almost everytime I'm softbanned because of this. However, I let my bot run for 1-2 hours before deciding to stop it.
